terminus/tests/road_test.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from geometry.point import Point
+from geometry.latlon import LatLon
+from models.road import Road
+from models.street import Street
+import math
+
+
+class RoadTest(unittest.TestCase):
+
+    def test_get_waypoint_positions(self):
+        points = [Point(0.0, 1.0), Point(1.0, 1.0), Point(2.0, 1.0)]
+        road = Street.from_points(points)
+        self.assertEqual(road.get_waypoint_positions(), points)
+
+    def test_get_waypoint_distances(self):
+        points = [Point(0.0, 1.0), Point(1.0, 1.0), Point(2.0, 1.0)]
```

Comment:
Same as before with the empty case. I would also test the case of a street with a single node. Finally, maybe having a distance different than 1.0 would be good.